index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Web Site</title>
+    <link rel="stylesheet" href="css/index.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat:200,200i,300,300i,400,600,600i,700,700i,800,800i,900,900i&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main id="main-container">
+      <header class="header">
+        <ul>
+          <li><a href="##">About</a></li>
+          <li><a href="##">Experience</a></li>
+          <li><a href="#">Skills</a></li>
+          <li><a href="#">Connect</a></li>
+        </ul>
+      </header>
+      <div id="container">
+        <section>
+          <h1>About</h1>
+          <p>
+            About me: My name is Abebe Tesso. I live in California. I am
+            originally from Ethiopia. I am happy to be a Code The Dream student
+            and eager to learn web development.
+          </p>
+        </section>
+        <section>
+          <h2>Experience</h2>
+          <p>
+            I have done HTML, CSS and JavaScript Projects with Code Tenderloin.
```

Comment:
I think they want the experience section to be a list just like the skills section.